ThirdParty/npm/pako.js

```diff
@@ -0,0 +1,2 @@
+import pako from 'pako';
+export { pako as default };
```

Comment:
If all we use is pako-inflate, would it be better to only include inflate here?  Then make sure roll-up treeshakes and it should create a smaller file.

Reply:
treeshaking may not work, but pako also includes an inflate only dist.. looks like we can save over a hundred KB by using that instead.

Reply:
This should be as simple as going back to inflate only like:

```
import pako_inflate from 'pako/lib/inflate.js';
export { pako_inflate as default };
```

and then update the calling code to just call `pako_inflate` instead of `pako.inflate`.

Reply:
Actually, I'm wrong.. you literally don't need to change the calling code.. just do

```
import pako from 'pako/lib/inflate.js';
export { pako as default };
```

and you end up with a `pako` object that only has inflate related stuff.